tests/PKSim.Tests/IntegrationTests/DatabaseUpdateSpecs.cs

```diff
@@ -241,6 +242,25 @@ public override void GlobalContext()
          _calculationMethodRepository = IoC.Resolve<ICalculationMethodRepository>();
          _rateFormulaRepository = IoC.Resolve<IRateFormulaRepository>();
          _rateObjectPathsRepository = IoC.Resolve<IRateObjectPathsRepository>();
+         _parameterDistributionRepository = IoC.Resolve<IParameterDistributionRepository>();
+      }
+
+      [Observation]
+      public void should_update_nhanes_values()
+      {
+         var nhanesParams = _parameterDistributionRepository.All().Where(pd => pd.Population.EndsWith("NHANES_1997")).ToList();
+
+         var bwParams = nhanesParams.Where(p => p.ParameterName.Equals(CoreConstants.Parameter.MEAN_WEIGHT)).ToList();
+         var heightParams = nhanesParams.Where(p => p.ParameterName.Equals(CoreConstants.Parameter.MEAN_HEIGHT)).ToList();
+         var volumeParams = nhanesParams.Where(p => p.ParameterName.Equals(CoreConstants.Parameter.VOLUME)).ToList();
+
+         //check number of new bw/height/volume parameters
+         (bwParams.Count()+ heightParams.Count()+ volumeParams.Count()).ShouldBeEqualTo(1420);
```

Comment:
you could remove all the () after Count since you are dealing with lists. 
Suggestion is to run the clean method from resharper